recipes/icu/all/conanfile.py

```diff
@@ -187,15 +183,6 @@ def generate(self):
     def _patch_sources(self):
         apply_conandata_patches(self)
 
-        if not self._with_unit_tests:
-            # Prevent any call to python during configuration, it's only needed for unit tests
-            replace_in_file(
-                self,
-                os.path.join(self.source_folder, "source", "configure"),
-                "if test -z \"$PYTHON\"",
-                "if true",
-            )
```

Comment:
Why should we remove this? This was here to prevent any call to the [Python test block](https://github.com/unicode-org/icu/blob/99e6053b78c355f4696276094e8fa954c67007af/icu4c/source/configure.ac#L1253), right?

Should we not keep that?

```python
    def _patch_sources(self):
        apply_conandata_patches(self)
        # Prevent any call to python during configuration, it's only needed for unit tests
        replace_in_file(
            self,
            os.path.join(self.source_folder, "source", "configure"),
            "if test -z \"$PYTHON\"",
            "if true",
        )
```